tokio-threadpool/src/pool/backup.rs

```diff
@@ -0,0 +1,280 @@
+use park::DefaultPark;
+use worker::{WorkerId};
+
+use std::cell::UnsafeCell;
+use std::fmt;
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering::{self, Acquire, AcqRel, Relaxed};
+
+#[derive(Debug)]
+pub(crate) struct Backup {
```

Comment:
What are `Backup`s responsibilities? Why is it called Backup?

Reply:
Done... hopefully it makes sense.

`Backup` isn't a great name though...

Reply:
thanks